src/accumulation/point_cloud_accumulator.py

```diff
@@ -76,3 +77,5 @@ def __load_instance_point_cloud(self,
                                         instance_id=instance_id,
                                         frame_point_cloud=frame_point_cloud,
                                         nuscenes=self.__nuscenes)
+
+        return instance_ptcl
```

Comment:
There are 2 **returns** in the same method. This is one is redundant

Reply:
typo after resolving! removed